styles.css

```diff
@@ -2,377 +2,571 @@
+  #resumeBtn {
+    font-size: 100%;
+    width: 50%;
+    padding: 5% 0;
   }
 
-  .socialsContainers {
-    color: #7f8cff;
-    font-weight: 500;
-    font-size: 16px;
-    line-height: 24px;
+  .aboutTitles {
+    width: 100%;
+    height: 190px;
   }
 
-  .socialIcons {
-    padding-right: 17px;
-    text-decoration: none;
-    display: inline;
+  #aboutSummary {
+    width: 70%;
   }
 
   /* WORK SECTION */
   .workSection {
     display: grid;
     grid-template-columns: auto;
     grid-row-gap: 88px;
-    padding: 114px 0;
+    padding: 154px 0;
     background-color: #e5e5e5;
     align-content: center;
     align-items: center;
   }
 
   .workSamples {
-    background-color: #fff;
-    border-radius: 16px;
-    padding: 16px;
-    padding-bottom: 30px;
-    display: flex;
-    flex-direction: column;
-    box-sizing: border-box;
-    width: 327px;
-    border: 1px solid #dfe1e6;
-    margin: 0 auto;
-  }
-
-  .workImages {
-    width: 295px;
-    height: 220px;
+    flex-direction: row;
+    width: 90%;
+    padding: 0 16px;
   }
 
   .workContents {
-    display: flex;
-    flex-direction: column;
+    padding: 2% 5%;
+    justify-content: start;
   }
 
   .workTitles {
     font-weight: 700;
-    font-size: 32px;
-    line-height: 35px;
-    color: #172b4d;
+    font-size: 20px;
+    line-height: 5px;
   }
 
   .workMetadatas {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
-    width: 80%;
-    height: 16px;
-    font-weight: 600;
-    font-size: 13px;
-    line-height: 16px;
-    color: #344563;
-  }
-
-  .inactiveCategories {
-    color: #7a869a;
+    width: 55%;
+    height: 10px;
+    font-size: 8px;
+    line-height: 10px;
   }
 
   .workDescriptions {
-    height: 76px;
-    font-weight: 400;
-    font-size: 15px;
-    line-height: 24px;
-    color: #344563;
+    height: auto;
+    font-size: 10px;
+    line-height: 18px;
   }
 
   .workCTAs {
-    display: flex;
-    flex-direction: column;
-    gap: 16px;
-    padding-top: 16px;
-    justify-items: center;
-  }
-
-  .workTags li {
-    text-decoration: none;
-    display: inline;
-    padding: 5px 12px;
-    height: 24px;
-    background: #ebebff;
-    border-radius: 8px;
+    padding: 0;
   }
 
-  a .workTags {
-    text-decoration: none;
-    font-weight: 500;
-    font-size: 12px;
-    line-height: 16px;
-    letter-spacing: 0.03em;
-    color: #6070ff;
+  .workTags {
+    font-size: 10px;
   }
 
-  a {
-    text-decoration: none;
-    color: #396df2;
+  .workTags li {
+    margin-right: 3%;
   }
 
   .workBtns {
-    padding: 26px;
-    background: #fff;
-    border: 1px solid #396df2;
-    border-radius: 12px;
-    width: 58%;
-    height: 24px;
-    font-weight: 500;
-    font-size: 17px;
-    line-height: 24px;
-    display: flex;
-    text-align: center;
-    align-items: center;
-    letter-spacing: 0.03em;
-    color: #396df2;
+    padding: 15px 0;
+    width: 37%;
+    font-size: 11px;
   }
```

Comment:
- [ ] Kindly consider adjusting the work section styles to match that of the `Figma` design template as much as possible. This will make your design look beautiful and create a good user experience. See below for more details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="1183" alt="Screenshot 2023-05-12 at 09 49 51" src="https://github.com/geekelo/a-personal-portfolio-site/assets/107192076/5336ef87-3a50-4914-9d0d-86f7dc2acc85">
</td>
<td>
<img width="1337" alt="Screenshot 2023-05-12 at 09 52 04" src="https://github.com/geekelo/a-personal-portfolio-site/assets/107192076/81f42955-aa6d-43a0-99dd-db5f1aaaedb3">
</td>
</tr>
</table>

- [ ] Also please note that on the desktop screen, some of the cards are `row-reversed` so please make sure you adhere to the details of the details. See below for more details:
See below for more details:
<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="1183" alt="Screenshot 2023-05-12 at 09 50 29" src="https://github.com/geekelo/a-personal-portfolio-site/assets/107192076/4abacdb1-306d-4c4a-b284-c80ab68983b1">
</td>
<td>
<img width="1337" alt="Screenshot 2023-05-12 at 09 52 16" src="https://github.com/geekelo/a-personal-portfolio-site/assets/107192076/f61dbe0d-52e5-4b2d-891d-a50dd509ab38">
</td>
</tr>
</table>